src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/heads/custom_fcn_mask_head.py

```diff
@@ -0,0 +1,83 @@
+"""CustomFCNMaskHead for OTX template."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numpy as np
+import torch
+from mmdet.models.roi_heads.mask_heads.fcn_mask_head import FCNMaskHead
+from mmdet.registry import MODELS
+from mmengine.config import ConfigDict
+from torch import Tensor
+
+
+@MODELS.register_module()
+class CustomFCNMaskHead(FCNMaskHead):
+    """Custom FCN Mask Head for fast mask evaluation."""
+
+    def _predict_by_feat_single(
+        self,
+        mask_preds: Tensor,
+        bboxes: Tensor,
+        labels: Tensor,
```

Comment:
These variabla names do not seem to be consistent. For masks, we use `preds` suffix, but not for `bboxes` and `labels`. In terms of consistency, would it be an idea to have something like:

```
pred_masks: Tensor,
pred_bboxes: tensor,
pred_labels: Tensor,
```